consensus/istanbul/backend/handler.go

```diff
@@ -287,8 +285,8 @@ func (sb *Backend) RegisterPeer(peer consensus.Peer, isProxiedPeer bool) {
 		}
 	}
 
-	if peer.Version() >= 65 {
-		sb.sendGetAnnounceVersions(peer)
+	if err := sb.sendAllSignedAnnounceVersions(peer); err != nil {
```

Comment:
With this change to unconditionally send the announce versions, we should probably also drop support for istanbul version 64 in the `eth` handshake.